python/ray/data/_internal/pipelined_dataset_iterator.py

```diff
@@ -19,13 +17,15 @@ def __init__(
     ):
         self._base_dataset_pipeline = base_dataset_pipeline
         self._epoch_iterator = None
+        self._to_tf_called = False
```

Comment:
I might have missed it, but why is this needed?

Reply:
ah sorry, good catch. I will remove this, it is not needed.